curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-do...while-loops.english.md

```diff
@@ -7,37 +7,35 @@ videoUrl: 'https://scrimba.com/c/cDqWGcp'
 
 ## Description
 <section id='description'>
-You can run the same code multiple times by using a loop.
-The next type of loop you will learn is called a "<code>do...while</code>" loop because it first will "<code>do</code>" one pass of the code inside the loop no matter what, and then it runs "<code>while</code>" a specified condition is true and stops once that condition is no longer true. Let's look at an example.
+The next type of loop you will learn is called a "<code>do...while</code>" loop.  It is called a <code>do...while</code> loop because it will first "<code>do</code>" one pass of the code inside the loop no matter what, and then continue to run the loop "<code>while</code>" the specified condition evaluates to <code>true</code>.
```

Comment:
```suggestion
The next type of loop you will learn is called a <code>do...while</code> loop.  It is called a <code>do...while</code> loop because it will first <code>do</code> one pass of the code inside the loop no matter what, and then continue to run the loop <code>while</code> the specified condition evaluates to <code>true</code>.
```
IMHO, this is fine too, the <code/> makes it different

Reply:
I agree, but several other guides have quotes around the name of the loop as well so I didn't change it in this PR.  I may go through and submit a new PR to remove quotes in many of the guides.

Reply:
That seems like a better option

Reply:
@thecodingaviator Are you ok merging as is?

Reply:
I guess so, @cmccormack can then go over all the other files having such issues or open an issue for a first timer kind of thing for such quotes to be removed. Looks fine to you @RandellDawson ?

Reply:
@thecodingaviator  Yes, it looks ok to me.  Feel free to create the issue regarding removing the quotes from these challenges.  Make sure to give a lot of detail (screenshot), so people know what and what not to remove.  We can mark it first timer and help wanted.